habitat-baselines/habitat_baselines/rl/hrl/hl/planner_policy.py

```diff
@@ -44,7 +44,7 @@ def __init__(self, *args, **kwargs):
 
     def apply_mask(self, mask):
         # We must replan.
-        self._should_replan *= mask.cpu().view(-1)
+        self._should_replan = ~mask.cpu().view(-1)
```

Comment:
should we make a flag of whether we should replan or not?

Reply:
This function call happens at every step or only when a skill is finished? If it happens at every step, I need to pass a different kind of mask to indicate that the skill is done.